src/genericComponents/Dropdown.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Col, Form } from 'react-bootstrap';
+// import { colors, fonts } from '../theme/styleVars';
+
+const DropdownMenu = (props: any) => {
+  const {
+    defaultFormValue,
+    controlId,
+    label,
+    onChange,
+    selectedValue,
+    values
+  } = props;
+  return (
+    <Form.Group as={Col} controlId={controlId}>
+      <Form.Label>{label}</Form.Label>
+      <Form.Control
+        as="select"
+        defaultValue={defaultFormValue}
+        onChange={onChange}
+      >
+        {values.map((value: string) => (
```

Comment:
Instead of just returning `value` here, you might actually want to return an object that has both name and value. Something like `{ name: string, value: string }`